CONTRIBUTING/adding_editing_information.md

```diff
@@ -59,103 +63,117 @@ git push
-- Click the button that reads _"New Pull Request"_
-- Click the text link near the top that says _"compare across forks"_
-- In the right-hand _"head repository"_ drop down, select your username / fork.
-- If you're happy with the list of commits shown, and the diff in the _"Files Changed"_ tab, fill in a title and description and click _"Create pull request"_
+- Go to your repository on GitHub and click "New Pull Request".
+- Alternatively, go to the main repository on GitHub ([https://github.com/ScilifelabDataCentre/pathogens-portal](https://github.com/ScilifelabDataCentre/pathogens-portal)) and click "New Pull Request", then click the text link near the top that says "compare across forks".
+- At the top of the page, you should see a pale grey box under the 'Comparing changes' text. The right-hand "head repository" drop down should show your username/fork and the branch that you worked on. Under 'base repository' you should see "ScilifelabDataCentre/pathogens-portal" and the 'base' should read 'develop'.
+- If you're happy with the list of commits and file changes shown towards the bottom of the page, click "Create pull request".
+- Write a title and description for your pull request, and then click 'Create pull request'.
+- A member of the portal team will review the pull request as quickly as possible regarding the pull request. Once approved, the changes will show on the portal.
 
-Once created, a member of the Portal team will review your changes.
-Once approved, they will be merged and published.
+## How to add a new page
 
-#### Testing with a local copy of the Portal
+The easiest way to add a new page to the portal is to [make a local copy](#using-the-web-interface). You can then navigate to the folder to which you would like to add a page (see below to gain an idea of where this might be, or check the URL of pages in the same section, as that will indicate the file structure). Create a new markdown (.md) file. We recommend copying the metadata from an existing page from the same section to initiate your page. The metadata is the piece at the top of the file that is between two lines of 3 dashes (i.e. '---'). You will have to change the metadata to reflect information about your new page, but this will ensure that your page is correctly initiated.
 
-Because the Portal is built on Hugo, it is quite easy to run a full version of the Portal on your computer and see how your changes look while doing that. See [this page for information on how to do that](https://github.com/ScilifelabDataCentre/pathogens-portal/blob/develop/CONTRIBUTING/running_a_local_copy.md).
+## Adding available data
 
-## Available datasets
+We have a database of available data/code from Sweden that can be found [in the available data section of the portal ](http://pathogens.se/datasets/all/). Multiple different types of data can be added to this database. Please note that this database is also updated approximtely monthly by the portal team.
```

Comment:
Can we use different word than "database", as (technical) people might think of an external database (like sql, etc)